### DISCUSSION TITLE
NOCODB + Digital Ocean Managed Database

### DISCUSSION THREAD

**Discussion by dopry** [nocodb/nocodb#4374] [Q&A] [answered]
I've tried setting up nocodb with a digital ocean managed database using an App.  I'm setting DATABASE_URL with the connection string provided by d.o, but it just doesn't seem to work no matter what I do. Has anyone else gotten noco working on D.O. with a managed database?

**Comment by dopry** [comment, marked as answer]
I finally figured this out... I had to set the env var `NC_DB_JSON` so I could embed the `ca` certificate downloaded from D.O.. You also need to use the `doadmin` user, it is the only user by default with permissions necessary to create the required schema. someone more familiar with postgres could probably write the grant for another user.  You also must set `rejectUnauthorized` to false to work with D.O. self signed certs I believe. 
```json
{
  "client": "pg",
  "connection": {
    "host": "yourhost.db.ondigitalocean.com",
    "port": "your port",
    "user": "doadmin",
    "password": "you password from d.o.",
    "database": "nocodb",
    "ssl": {
      "ca": "-----BEGIN CERTIFICATE-----\n.....\n..... \n....\n-----END CERTIFICATE-----",
      "key": "",
      "cert": "",
      "rejectUnauthorized": false
    }
  }
}
```

**Comment by mertmit** [comment]
Happy to hear you managed to get it work. For future reference:
We have [1-click deploy on DigitalOcean](https://github.com/nocodb/nocodb-seed-heroku) with many others as well.

**Reply by dopry** [reply]
I see for the one-click you add ssl=no-verify option... It would be nice if all the options for the settings and connection strings were documented.
